sdk/keyvault/azure-keyvault-keys/tests/test_key_client.py

```diff
@@ -359,3 +362,73 @@ def test_purge(self, vault_client, **kwargs):
         # validate none are returned by list_deleted_keys
         deleted = [s.name for s in client.list_deleted_keys()]
         self.assertTrue(not any(s in deleted for s in key_names))
+
+    @ResourceGroupPreparer(name_prefix=name_prefix)
+    @VaultClientPreparer(client_kwargs={'logging_enable': True})
+    def test_logging_enabled(self, vault_client, **kwargs):
+        class MockHandler(logging.Handler):
+            def __init__(self):
+                super(MockHandler, self).__init__()
+                self.messages = []
+            def reset(self):
```

Comment:
`reset` isn't part of the `Handler` contract and we never call it, so we may as well not define it.

Reply:
ok, getting rid of it